libcaf_core/caf/event_based_actor.test.cpp

```diff
@@ -29,7 +29,21 @@ TEST("unexpected messages result in an error by default") {
     };
   });
   expect<std::string>().from(sender).to(receiver);
-  expect<error>().with(sec::unexpected_message).from(receiver).to(sender);
+  SECTION("receiver sends an unexpected_message error to sender") {
+    expect<error>().with(sec::unexpected_message).from(receiver).to(sender);
+  }
```

Comment:
This section is a single line that is also in the other section. I'd drop the sections entirely.